.github/workflows/ci.yml

```diff
@@ -373,6 +373,11 @@ jobs:
             #
             #          cargo test -p polygon-pos-prover -- --nocapture --ignored
 
+
+            - name: polygon pos integration tests
+              run: |
+                  RUST_LOG=tesseract=trace, cargo test -p tesseract-integration-tests -- --nocapture
```

Comment:
Just add a polygon mainnet test to tendermint prover, tesseract tests will not work in CI